tasks/gene_typing/task_abricate.wdl

```diff
@@ -60,13 +60,13 @@ task abricate_flu {
     File assembly
     String samplename
     String database = "insaflu"
-    String nextclade_flu_h1n1_ha_tag
-    String nextclade_flu_h1n1_na_tag
-    String nextclade_flu_h3n2_ha_tag
-    String nextclade_flu_h3n2_na_tag
-    String nextclade_flu_vic_ha_tag
-    String nextclade_flu_vic_na_tag
-    String nextclade_flu_yam_tag
+    String? nextclade_flu_h1n1_ha_tag
+    String? nextclade_flu_h1n1_na_tag
+    String? nextclade_flu_h3n2_ha_tag
+    String? nextclade_flu_h3n2_na_tag
+    String? nextclade_flu_vic_ha_tag
+    String? nextclade_flu_vic_na_tag
+    String? nextclade_flu_yam_tag
```

Comment:
Without a subtype, how will this impact nextclade downstream?

Reply:
Without a subtype, Nextclade will not run